server/server.go

```diff
@@ -946,3 +948,80 @@ func isNotFound(err error) bool {
 
 	return false
 }
+
+// startWatcherForMirrorRegistries sets up a file watcher to monitor changes
+// in the "registries.conf.d" directory (default: "/etc/containers/registries.conf.d").
+// It then delegates the monitoring task to watchAndReloadMirrorRegistriesConfiguration,
```

Comment:
Perhaps? Unless there is something missing there and would continue in the next line.

```suggestion
// It then delegates the monitoring task to watchAndReloadMirrorRegistriesConfiguration.
```